### ISSUE TITLE
Rendering Footnotes

### ISSUE THREAD

**Issue by stoweboyd** [mcndt/obsidian-quickshare#11]
It would be good if footnotes were rendered in the copied file.

In my use case, I write comments and place tags and other annotations in footnotes, but for sharing with others those would be better at the foot of the rendered file.

**Comment by mcndt** [comment]
I agree, footnotes should be rendered correctly. I'll mark it as a future feature. Stay tuned!
